sympy/functions/elementary/tests/test_piecewise.py

```diff
@@ -1136,5 +1136,7 @@ def test_issue_8458():
     x, y = symbols('x y')
     p1 = Piecewise((0, x == 0), (sin(x), True))
```

Comment:
This should be `Eq(x, 0)` since `x==0` is automatically False and p1 is already `sin(x)` before simplification.

Reply:
Of course! Still improving my basics... Thanks!